Source/Core/DolphinQt2/GameList/GameFileCache.h

```diff
@@ -26,5 +26,4 @@ class GameFileCache
   QString m_path;
 
   QMap<QString, GameFile> m_gamefiles;
-  std::mutex m_mutex;
```

Comment:
The include for `<mutex>` can be removed.

Reply:
Done.